src/lib/learn-client/api/bookmark/get-bookmarks.ts

```diff
@@ -0,0 +1,49 @@
+import { get, toError } from 'lib/learn-client'
+import { SessionData } from 'types/auth'
+import { BOOKMARK_API_ROUTE } from '.'
+import { ApiBookmark, ApiTutorial } from '../api-types'
+
+interface GetBookmarksOptions {
+	accessToken: SessionData['accessToken']
+	tutorialIds?: ApiTutorial['id'][]
+}
+
+type GetBookmarksResult = ApiBookmark[]
+
+/**
+ * Fetches bookmarks based on given `tutorialIds`. If none are provided, then
+ * all bookmarks will be fetched. If some `tutorialIds` are provided, then they
+ * will be passed in the comma-separated `tutorialIds` query parameter.
+ *
+ * If successful, then an array of bookmark objects is returned. Otherwise
+ * throws an error.
+ *
+ * https://digital-api-specs.vercel.app/learn#tag/Bookmarks/paths/~1bookmarks/get
+ */
+const getBookmarks = async ({
+	accessToken,
+	tutorialIds,
+}: GetBookmarksOptions): Promise<GetBookmarksResult> => {
+	// Build the API route to make a request to
+	let requestRoute = BOOKMARK_API_ROUTE
+	if (tutorialIds && tutorialIds.length > 0) {
+		const commaSeparatedIds = tutorialIds?.join(',')
```

Comment:
```suggestion
		const commaSeparatedIds = tutorialIds.join(',')
```

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/828/commits/16768a7dfde1e4ef1f3232abbf20a12456405aaa!